browser/brave_content_browser_client.cc

```diff
@@ -391,6 +392,34 @@ bool BraveContentBrowserClient::BindAssociatedReceiverFromFrame(
   return false;
 }
 
+bool BraveContentBrowserClient::AllowWorkerFingerprinting(
+    const GURL& url,
+    content::BrowserContext* browser_context,
+    const std::vector<content::GlobalRenderFrameHostId>& render_frames) {
+  return WorkerGetBraveFarblingLevel(url, browser_context, render_frames) !=
+         BraveFarblingLevel::MAXIMUM;
+}
+
+uint8_t BraveContentBrowserClient::WorkerGetBraveFarblingLevel(
+    const GURL& url,
+    content::BrowserContext* browser_context,
+    const std::vector<content::GlobalRenderFrameHostId>& render_frames) {
```

Comment:
why pass `render_frames` at all?

Reply:
Fixed in https://github.com/brave/brave-core/pull/11452/commits/2528d59df430db0554d5c51c14dd3df1cfa72585